static/app/views/performance/browser/webVitals/utils/queries/storedScoreQueries/calculatePerformanceScoreFromStored.tsx

```diff
@@ -34,6 +39,11 @@ export function getWebVitalScores(data?: TableDataRow): ProjectScore {
     ttfbScore: Math.round(getWebVitalScore(data, 'ttfb') * 100),
     fidScore: Math.round(getWebVitalScore(data, 'fid') * 100),
     totalScore: Math.round(getTotalScore(data) * 100),
+    lcpWeight: Math.round(getWebVitalWeight(data, 'lcp') * 100),
+    fcpWeight: Math.round(getWebVitalWeight(data, 'fcp') * 100),
+    clsWeight: Math.round(getWebVitalWeight(data, 'cls') * 100),
+    ttfbWeight: Math.round(getWebVitalWeight(data, 'ttfb') * 100),
+    fidWeight: Math.round(getWebVitalWeight(data, 'fid') * 100),
```

Comment:
Have we ever seen these outside of `0,1`? We may want to clamp or send an exception and let it happen if we see it.

Reply:
Personally haven't seen values outside of 0 - 1, but I've added a check in `getWebVitalWeight` to throw an error if the weight is out of bounds.